server/src/main/java/org/opensearch/index/search/QueryStringQueryParser.java

```diff
@@ -470,15 +469,6 @@ private Query getRangeQuerySingle(
         }
     }
 
```

Comment:
This change would clearly break things :( (I believe)

Reply:
Since standard QueryParser is the most commonly used parse I have updated the code to extent the `extends QueryParser`, I can revert this and bring back the `server/src/main/java/org/apache/lucene/queryparser/classic/XQueryParser.java`.  Let me know.

Reply:
I see the Gradle check test `org.opensearch.index.query.QueryStringQueryBuilderTests.testToQueryFuzzyQueryAutoFuzziness` failing because of this change.

Reply:
Exactly

Reply:
Yeah... in OpenSearch, we have "richer" handling of fuzzy queries, beyond what the "classic" query parser does.

As the comment on `XQueryParser` says, we should perhaps consider using the `flexible` query parser implementation to properly customize fuzzy nodes. (I believe we would need to start with something like the `StandardQueryParser`, but override the implementation of `FuzzyQueryNodeProcessor`.)

It's a more involved change, though, which will definitely take more effort.

Reply:
Got It, then for now I will keep the `org.apache.lucene.queryparser.classic` package and `XQueryParser` class.

Reply:
The tests pass now, @reta @msfroh.

> As the comment on `XQueryParser` says, we should perhaps consider using the `flexible` query parser implementation to properly customize fuzzy nodes. (I believe we would need to start with something like the `StandardQueryParser`, but override the implementation of `FuzzyQueryNodeProcessor`.)

I can open an issue to track this.